### ISSUE TITLE
csv2-main crashed without visible traceback

### ISSUE THREAD

**Issue by rseuster** [hep-gc/cloudscheduler#137]
Here are the lines from the logfile, look at the large gap in timestamps. The process crashed until I restarted it 1 1/2 hours later:
```
2019-01-07 20:24:02,343 - MainProcess      - DEBUG - ---------------got idle jobs--------------
2019-01-07 20:24:02,417 - MainProcess      - DEBUG - Dealing with current group: test
2019-01-07 20:24:02,830 - MainProcess      - DEBUG - Added all clouds for group: test
2019-01-07 20:24:02,831 - MainProcess      - DEBUG - Check Idle Machines for Keep Alive time for: test
2019-01-07 21:52:32,117 - MainProcess      - DEBUG - Dealing with current group: atlas
2019-01-07 21:52:32,570 - MainProcess      - DEBUG - Added all clouds for group: atlas
```

whenever automated restart is enabled, we should first copy the last 50 lines of the logfile somewhere where we can check later.

**Comment by mhpx** [comment]
@colsond 
csmain is now using the process monitor class like the other pollers. Does anything else need to be done for this issue?

**Comment by colsond** [comment]
@mhpx 
The process monitor has nothing to do with the csstatus but it does log when something has died and when it restarts a process.

The status poller (csstatus.py) just uses an os call to do a systemctl status <service> and dumps the result into the database. In theory some data processing could be done there to determine if there was a crash and log an error for logstash. That may be redundant tho since the crashes are logged in the processes themselves. In it's current state the only thing the status poller logs is if it experiences a failure.

Long story short, if logstash is monitoring the other logs the crashes will be visible, if not- someone needs to upgrade the status poller to log based on the os command.

**Comment by colsond** [comment]
Looks like i lied a bit, the status poller does log a line if a service is found dead:
`logging.error("Found service %s is dead...", service)`

**Comment by mhpx** [comment]
basic feature in place. should we add configuration options to enable/disable or change the number of lines captured?

**Comment by rseuster** [comment]
fixed.
